.claude/scripts/task_impl.py

```
@@ -0,0 +1,43 @@
+    return a - b
+
+
+def multiply(a, b):
+    """Return the product of a and b."""
+    return a * b
+
+
+def divide(a, b):
+    """Return the quotient of a divided by b. Raises ZeroDivisionError if b is zero."""
+    if b == 0:
+        raise ZeroDivisionError("division by zero")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)